src/sapphire_engine.hpp

```diff
@@ -7,16 +7,21 @@
 #include <vector>
 #include <stdexcept>
 
+#ifdef NO_RACK_DEPENDENCY
 /*
  * In the rack context this ifdef isn't needed; rack gives you simde for free
  * but if you want to use this module outside of rack (which we do) you need to
  * have a bring-your-own simde approach which will just use RACK SIMDE in rack builds
  */
-#if defined(__SSE2__)
-#include <pmmintrin.h>
-#else
+#if defined(__arm64)
 #define SIMDE_ENABLE_NATIVE_ALIASES
 #include "simde/x86/sse4.2.h"
+#else
+#include <pmmintrin.h>
+#endif
+
+#else
+#include "rack.hpp"
```

Comment:
I don't think we need to include rack.hpp here at all. I believe including it from my plugin.hpp is all I need.

Reply:
I'm going to merge the PR, then delete the include. Please let me know if you have any concerns about that. I think it will be OK.

Reply:
OK, never mind. I see it was needed after all. I did have to make another change though that I pushed: adding `-D NO_RACK_DEPENDENCY` to a couple of build scripts for my own unit tests. Thanks @baconpaul! I'm handing the ball back to you now.

Reply:
Awesome. Updating the plugin now.

Yeah rack.hpp is how you get a portable ref to pmmintrin etc in rack land so if you want non-fragile headers you need *something* at that point.

Reply:
I should know by now to test something before I open my big mouth about it! LOL.

Reply:
ha!